src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+interface Props {
+  id: string;
+  isTile?: boolean;
+  label: string;
+  name: string;
+  tileDescription?: string;
+  value?: string;
+}
```

Comment:
I'm thinking we might need a children prop here, there's a few examples in the figma. Just like in QMR, we might need to click a checkbox that opens up a text input or whatever the user might need.

Reply:
Oh, of course! Great catch. I'll add that.